lib/mayaUsd/render/vp2RenderDelegate/proxyRenderDelegate.cpp

```diff
@@ -836,21 +868,28 @@ void ProxyRenderDelegate::_DirtyUsdSubtree(const UsdPrim& prim)
         | HdChangeTracker::DirtyDisplayStyle | MayaUsdRPrim::DirtySelectionHighlight
         | HdChangeTracker::DirtyMaterialId;
 
-    if (prim.IsA<UsdGeomGprim>() && prim.IsActive()) {
-        auto indexPath = _sceneDelegate->ConvertCachePathToIndexPath(prim.GetPath());
-        if (_renderIndex->HasRprim(indexPath)) {
-            changeTracker.MarkRprimDirty(indexPath, dirtyBits);
+    auto markRprimDirty = [this, &changeTracker, dirtyBits](const UsdPrim& prim) {
+        if (prim.IsA<UsdGeomGprim>()) {
+            if (prim.IsInstanceProxy()) {
```

Comment:
I see the old code was also checking IsActive() but this new code does not. IDK what is the implication of that change.

Reply:
There is no implication because of that since the call to _renderIndex->HasRprim will reject non-active prims